### ISSUE TITLE
Poster Design

### ISSUE THREAD

**Issue by jennykim1016** [jennykim1016/SLRealizer#58]
The deadline for department-wise poster printing is next Tuesday, 8/29 @ 2pm. I started to make poster in the branch `poster`, and I should finish the first draft by Thursday morning to go to the Thursday afternoon poster workshop :)

**Comment by drphilmarshall** [comment]
Great! The numbers are important to get right, in such a concise format:
LSST is an 8.4m telescope (not 4.8), and will produce a lot more than 30Tb
of data!! You could quote "30 trillion measurements of  37 billion objects"
from the LSST website if you like. You should explain the choice of 200
epochs simulated. This is about 2.5 years of LSST surveying - an integer
number of years would make more sense, given that LSST will release
catalogs annually. Could we switch to 250 epochs, or even better, all
epochs in the first 3 years?

If I were you I'd drop the magnitudes corner plot, and focus on the most
discriminating features. Did you get a chance to look at variations in
centroid position with color?

**Comment by jennykim1016** [comment]
Hi @drphilmarshall ,

Thank you so much for the feedback! I changed the code to choose first 268 epochs to include full three years of the survey. The first night seemed to be 59823.286523, and I queried the epochs that have MJD < 59823.286523+365.2425*3.

The variations in the centroid position with the color unfortunately did not provide useful information. 
![centroid_x](https://user-images.githubusercontent.com/18297066/29634862-0f2223e4-8800-11e7-98c7-356e050db371.png)
[centroid position along the x-axis]

![centroid_y](https://user-images.githubusercontent.com/18297066/29634877-191de860-8800-11e7-9701-84c2d4a3b5b9.png)
[centroid position along the y-axis]

![centroid_sum](https://user-images.githubusercontent.com/18297066/29634904-27ddcd8e-8800-11e7-878f-51455d3e3c15.png)
[centroid position(total) calculated by pythagorean theorem]

The most distinctive feature was in fact $u-z$ magnitude, and sum of the ellipticities was also interesting. I will replace the figures that show the most interesting features in the poster :)
